README.md

```diff
@@ -390,4 +400,4 @@ workflow, as it is intended to be very versatile.
 
 ## License
 
-bump2version is licensed under the MIT License - see the LICENSE.rst file for details
+bump2version is licensed under the MIT License - see the [LICENSE.rst](LICENSE.rst) file for details
```

Comment:
My only question is whether this will render well on [pypi](https://pypi.org/project/bump2version/).

Reply:
I think it will render well on PyPI because I just copied how the `CONTRIBUTING.md` is linked  (9 lines above), and it does render well on https://pypi.org/project/bump2version/

Reply:
Actually you are right, PyPI transforms the `CONTRIBUTING.md` link into https://pypi.org/project/bump2version/CONTRIBUTING.md, which is not a valid link...

I guess the options are:
1. accepting having broken links on PyPI
2. changing the links to absolute links (e.g.: `[LICENSE.rst](https://github.com/c4urself/bump2version/blob/master/LICENSE.rst)`)

Reply:
Given it's already an issue with the other guide and it's not a real issue, I'm ok with option 1 and accept the broken link on PyPI.